### ISSUE TITLE
Training-related issues

### ISSUE THREAD

**Issue by zzzcyyyw** [tapos-datta/HumanSegmentation#1]
Hi, can I ask you the information about the graphics card you used to run this project and how many epochs you ran?
Did you keep the learning rate constant during the training process?

**Comment by tapos-datta** [comment]
Hello, 
I ran the project with nvidia RTX-4090 and used a constant learning rate (i.e. 0.0009) during the whole training process (440 epochs) with batch size 20.
